server/models/contact.coffee

```diff
@@ -28,6 +28,14 @@ module.exports = class Contact extends cozydb.CozyModel
         # vCard Name = splitted
         # (Familly;Given;Middle;Prefix;Suffix)
         n             : String
+        org           : String
+        title         : String
+        department    : String
+        bday          : String
+        nickname      : String
+        url           : String
+        import        : Boolean
+        rev           : Date
```

Comment:
Just extend it :

class Contact extends cozydb.Model
     @schema: 
            ...
     
     updateAttributes: (changes, callback) ->
          changes.rev = Date.now().toISOString()
          super